File: pset6/sentimentals/mario.py
from cs50 import get_int

def main():

    # Do-while loop to prompt user for an input (Positive Number)
    while True:
        height = get_int("Positive Number: ")

        # Ensures input is a positive number
        if height >= 0 and height <= 23:
            break

    for rows in range(0, height):

        # Spaces before pyramid
        for spaces in range(1, height - rows):
            print(" ", end ="")

        # For left pyramid
        for left_hashses in range(height - rows, height + 1):
            print("#", end ="")

        # For gap
        print ("  ", end ="")

        # For right pyramid
        for right_hashes in range(height - rows, height + 1):
            print ("#", end ="")

        # Print newline
        print()

if __name__ == "__main__":
    main()

File: pset1/mario/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // Prompt user for an input (Positive Number)
    int n;
    do
    {
        n = get_int("Positive Number: ");
    }
    while (n < 0 || n > 23);

    for (int i = 0; i < n; i++)
    {
        // Spaces before pyramid
        for (int j = n - i - 1; j > 0; j--)
        {
            printf(" ");
        }
        // For Left Pyramid
        for (int k = 0; k < i + 1; k++)
        {
            printf("#");
        }
        // For Gap
        {
            printf("  ");
        }
        // For Right Pyramid
        for (int m = 0; m < i + 1; m++)
        {
            printf("#");
        }
        printf("\n");
    }
}
